client/src/components/Input/Input.tsx

```diff
@@ -15,6 +22,8 @@ const Input: FC<InputHTMLAttributes<HTMLInputElement> & InputProps> = ({
   value,
   onChange
 }) => {
+  const { darkMode } = useContext(DarkLightThemeContext);
```

Comment:
it's not related to redux, next time please select files for pushing attentively